docsite/index.html

```diff
@@ -109,13 +109,13 @@ <h1><span>CSS</span> <span>variables.</span></h1>
             <svg viewBox="0 0 24 24">
               <use href="#checkmark-icon"/>
             </svg>
-            Create consistent components
+            Consistent styles and conventions
```

Comment:
Mainly changed this for parallelism so it's consistent with the other bullets, feel free to propose an alternative!

Reply:
thanks for offering a suggestion here, but i prefer how it was..!